kimchi/src/circuits/polynomials/rot.rs

```diff
@@ -328,8 +328,6 @@ pub fn extend_rot<F: PrimeField>(
     rot: u32,
     side: RotMode,
 ) {
-    assert!(rot < 64, "Rotation value must be less than 64");
```

Comment:
Don't you want `assert!(rot <= 64,` instead?

Reply:
Oh true, thanks for the nit, I missed that line!